### ISSUE TITLE
Integration test does not start

### ISSUE THREAD

**Issue by tomjirsa** [CSIRT-MU/Stream4Flow#43]
*  After a clean installation - screen does not start the script. Ends in
```
spark@sparkMaster:~$ screen -list
There are screens on:
	14305..sparkMaster	(05/23/2017 10:23:31 AM)	(Dead ???)
	14197..sparkMaster	(05/23/2017 10:23:31 AM)	(Dead ???)
```

* I would increase waiting times for application start and data processing - the data are read after second try

**Comment by simkosvorad** [comment]
I slightly increansed witing times - for application start 25s , for data processing 35s. I also changed time deffinitions to variables so it could be easily adjusted.

**Comment by tomjirsa** [comment]
see comment in pull request #48

**Comment by tomjirsa** [comment]
Closed by merge #48
